tests/test_on_data/test_gin_ecephys.py

```diff
@@ -448,7 +451,16 @@ class TestConverter(NWBConverter):
             # NWBSortingExtractor on spikeinterface does not yet support loading data written from multiple segment.
             if sorting.get_num_segments() == 1:
                 nwb_sorting = NwbSortingExtractorSI(file_path=nwbfile_path, sampling_frequency=sf)
-                check_sorting_equal_si(SX1=sorting, SX2=nwb_sorting)
+
+                ids1 = np.sort(np.array(sorting.get_unit_ids()))
+                ids2 = np.sort(np.array(nwb_sorting.get_unit_ids()))
+                if np.allclose(ids1, ids2):
+                    check_sorting_equal_si(SX1=sorting, SX2=nwb_sorting)
```

Comment:
Ends up breaking a NeuroScope test if you just always do the renaming thing even if they already match... not sure why that doesn't break on https://github.com/catalystneuro/neuroconv/pull/255 though